packages/label-sync/src/label-sync.ts

```diff
@@ -0,0 +1,176 @@
+/**
+ * Copyright 2019 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Application } from 'probot';
+import { request } from 'gaxios';
+import { GitHubAPI } from 'probot/lib/github';
+
+interface Labels {
+  labels: [
+    {
+      name: string;
+      description: string;
+      color: string;
+    }
+  ];
+}
+
+interface Repos {
+  repos: [
+    {
+      language: string;
+      repo: string;
+    }
+  ];
+}
+
+// Labels are fetched by reaching out to GitHub *instead* of grabbing the file
+// from the local copy.  We are using the `PushEvent` to detect the change,
+// meaning the file running in cloud will be older than the one on master.
+let labelsCache: Labels;
+
+async function getLabels() {
+  if (!labelsCache) {
+    await refreshLabels();
+  }
+  return labelsCache;
+}
+
+async function refreshLabels() {
+  const url =
+    'https://github.com/googleapis/repo-automation-bots/blob/master/packages/label-sync/src/labels.json';
+  const res = await request<Labels>({ url });
+  labelsCache = res.data;
+}
+
+export = (app: Application) => {
+  const events = [
+    'repository.created',
+    'repository.transferred',
+    'label.edited',
+    'label.deleted',
+  ];
+
+  events.forEach(evt =>
+    app.on(evt, async c => {
```

Comment:
you can provide an array to `events` rather than calling `app.on` multiple times I believe.